ncurses.go

```diff
@@ -253,6 +253,11 @@ func ResizeTerm(nlines, ncols int) error {
 	return nil
 }
 
+// Set the ESCDELAY
```

Comment:
A slightly more detailed description would be helpful. This is an area of goncurses that could definitely be improved on. No need to go into much detail or do a copy and paste. Just slightly more detail.

Example: `Sets the delay from when the escape key is pressed until ...`